spec/models/learning_unit_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe LearningUnit, type: :model do
+  it 'is not valid without a name' do
```

Comment:
En gral no entiendo por qué testearías algo que es condición en tu model, ¿cuándo no se cumpliría que tiene nombre?

Reply:
No sé. Seguí guías de internet (https://semaphoreci.com/community/tutorials/how-to-test-rails-models-with-rspec). Probablemente para evitar a futuro que se rompan cosas.

Reply:
Eliminé el test.